### PR TITLE
1856 rights nationalization

### PR DESCRIPTION
**Pull request by ventusignis**
The tunnel & bridge rights get transferred to the national during nationalization; duplicates get popped to the bank pool where other corporations may purchase them

### CODE REVIEW

File: lib/engine/step/g_18_sj/special_buy.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@ def process_special_buy(action)
 
         def setup
           super
-          @gkb_bonus_item = Item.new(description: 'GKB bonus', cost: 0)
+          @gkb_bonus_item ||= Item.new(description: 'GKB bonus', cost: 0)
```
Comment (tobymao) [inline lib/engine/step/g_18_sj/special_buy.rb:30]: why is this changed?

Comment (ventusignis) [reply lib/engine/step/g_18_sj/special_buy.rb:30]: This was supposed to be in the prior PR but the push from my working copy failed due to a typo

Comment (tobymao) [reply lib/engine/step/g_18_sj/special_buy.rb:30]: is this needed?

Comment (ventusignis) [reply lib/engine/step/g_18_sj/special_buy.rb:30]: It will work as-is. This change will prevent a new instance of the Item being created every operating round

**Review by tobymao — APPROVED** [review]
